c++/src/ColumnReader.cc

```diff
@@ -581,6 +581,8 @@ namespace orc {
     lengthDecoder->next(lengthArray + 1, dictSize, nullptr);
     lengthArray[0] = 0;
     for(uint32_t i = 1; i < dictSize + 1; ++i) {
+      if (lengthArray[i] < 0)
```

Comment:
Please add curly brace here even for a single line if-statement.

Reply:
Done